### PR TITLE
Feat(frontend): Rename and modify the 'PhoneBrand' component to be more general

### PR DESCRIPTION
**Pull request by GuillaumeKESTEMAN**
Rename the component : 'PhoneBrand' to 'Select' 
Modifications : make this component more general, now it's a general component which needs a title and an array of options

### CODE REVIEW

**Review by nfroidure — CHANGES_REQUESTED** [review]
Otherwise LGTM

File: front/components/Select.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        }: Omit<SelectHTMLAttributes<HTMLSelectElement>, "title"> & Omit<SelectHTMLAttributes<HTMLOptionElement>, "options"> & {
+            title: string;
+            options: Array<string>;
+        },
+        ref?: LegacyRef<HTMLSelectElement>
+    ) => {
+    return (
+      <select
+        ref={ref}
+        name="brand"
```
[the remark this answers is not in the document]
Comment (GuillaumeKESTEMAN) [reply front/components/Select.tsx]: This has been added :+1:

**Review by nfroidure — APPROVED** [review]
LGTM

**Comment by nfroidure** [comment]
Just squash the 2 commits into one and you're done ;). Congratz!

**Pull request merged**